mkdocs/__main__.py

```diff
@@ -5,11 +5,6 @@
 import logging
 import click
 
-# TODO: Remove this check at some point in the future.
-# (also remove flake8's 'ignore E402' comments below)
```

Comment:
As we are removing this block, please also follow the advice on this line and remove the `# nova: E402` comments from the import lines below this block. They are no longer needed as this block is no longer before them. Otherwise, looks good.

Reply:
Done.